content/docs/user-guide/experiment-management/index.md

```diff
@@ -1,110 +0,0 @@
-  manage in batches.
-
-[run and capture]: /doc/user-guide/experiment-management/running-experiments
-[experiments]: /doc/user-guide/experiment-management/experiments-overview
-[checkpoints]: /doc/user-guide/experiment-management/checkpoints
-
-<admon type="">
-
-See 👨‍💻 [Get Started: Experiments] for a hands-on introduction to DVC
-experiments.
-
-</admon>
-
-[get started: experiments]: /doc/start/experiment-management/experiments
-
-### Organization patterns
-
-It's up to you to decide how to organize completed experiments. These are the
-main alternatives:
-
-- **Git tags and branches** - use the repo's "time dimension" to distribute your
-  experiments. This makes the most sense for experiments that build on each
-  other. Git-based experiment structures are especially helpful along with Git
-  history exploration tools
-  [like GitHub](https://docs.github.com/en/github/visualizing-repository-data-with-graphs/viewing-a-repositorys-network).
-
-- **Directories** - the project's "space dimension" can be structured with
-  directories (folders) to organize experiments. Useful when you want to see all
-  your experiments at the same time (without switching versions) by just
-  exploring the file system.
-
-- **Hybrid** - combining an intuitive directory structure with a good repo
-  branching strategy tends to be the best option for complex projects.
-  Completely independent experiments live in separate directories (and can be
-  generated with [`foreach` stages], for example), while their progress can be
-  found in different branches.
-
-- **Labels** - in general, you can record experiments in a separate system and
-  structure them using custom labeling. This is typical in dedicated experiment
-  tracking tools. A possible problem with this approach is that it's easy to
-  lose the connection between your project history and the experiments logged.
-
-DVC takes care of arranging `dvc exp` experiments and the data
-<abbr>cache</abbr> under the hood so there's no need to decide on the above
-until your experiments are made [persistent].
-
-[`foreach` stages]:
-  /doc/user-guide/project-structure/dvcyaml-files#foreach-stages
-[persistent]: /doc/user-guide/experiment-management/persisting-experiments
-
-## Run Cache: Automatic Log of Stage Runs
-
-Every time you [reproduce](/doc/command-reference/repro) a pipeline with DVC, it
-logs the unique signature of each stage run (in `.dvc/cache/runs` by default).
-If it never happened before, its command(s) are executed normally. Every
-subsequent time a [stage](/doc/command-reference/run) runs under the same
-conditions, the previous results can be restored instantly, without wasting time
-or computing resources.
-
-✅ This built-in feature is called <abbr>run-cache</abbr> and it can
-dramatically improve performance. It's enabled out-of-the-box (can be disabled),
-which means DVC is already saving all of your tests and experiments behind the
-scene. But there's no easy way to explore it.
```

Comment:
Minor: I'd maybe preserve some of this wording in https://dvc.org/doc/user-guide/project-structure/internal-files#run-cache (replacing some of the text there, which is older).

Reply:
Preserving this somewhere makes sense, but I don't see it as a core part of experiment management. What do you think about moving it to the pipelines guide?

Reply:
Will address in a separate PR